lib/fetch-data.js

```diff
@@ -124,42 +122,54 @@ class FetchData {
     }, done);
   }
 
-  getApi(apis, request, done) {
-    const mapObj = {};
-    if (Array.isArray(apis)) {
-      apis.forEach((api) => {
-        if (typeof api === 'string') {
-          mapObj[api] = api;
-        } else {
-          mapObj[api.url] = api;
+  getApi(apis, request, allDone) {
```

Comment:
why aren't we just using the map helper that all of the other functions are using?